### PR TITLE
PEM - 9.6 Release branch

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Added new alert templates as per https://github.com/EnterpriseDB/pem/pull/389 and https://github.com/EnterpriseDB/pem/pull/388

Started drafting release notes

### CODE REVIEW

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 1]
```diff
@@ -256,6 +259,52 @@ If you specify a parent node in the **Copy Probe Configuration** tree, PEM copie
 
 After you select the target object or objects, select **Configure Probes** to copy the probe definitions to the location selected in the dialog box.
 
+## Managing probes using pemworker
+
+You can enable and disable a probe using the pemworker utility from version 9.6 and later.
```
Comment (edbshubham) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:264]: Add the option here, the statment could be like this:-
You can enable and disable a probe using the pemworker utility from version 9.6 and later either by --probe-display-name, --probe-internal-name or by --probe-id

File: product_docs/docs/pem/9/monitoring_performance/probes.mdx [hunk 2]
```diff
@@ -73,6 +73,9 @@ Unless otherwise noted, Postgres Enterprise Manager enables the following probes
...
 | xDB Replication                          | Lag data for clusters replicated using xDB replication.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Database |
+| Multixact ID Exhaustion (Wraparound)     | Information about the maximum transaction ID. The stored information includes:<br/>The name of the database<br/>The oldest current Multixact ID in the database<br/>Percentage towards wraparound<br/>Percentage towards emergency autovacuum   | Server |
```
Comment (edbshubham) [inline product_docs/docs/pem/9/monitoring_performance/probes.mdx:76]: still maximum transaction id is written

File: product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+| Bug&nbsp;fix | Fixed an issue where audit manager was setting `edb_audit_statement` parameter to '' instead on 'none' when log statement parameter was left empty in the GUI resulting in failure of server restart.                                               |
+| Bug&nbsp;fix | Updated the default parameter for configuring PEM to use the `Fully Qualified Domain Name (FQDN)` hostname while agent registration.                                                                                                                |
+| Bug&nbsp;fix | Fixed an issue with error `NoneType` has no `len()` when alerts were listed on the alert dashboard.                                                                                                                                                 |
+| Bug&nbsp;fix | Fixed an issue for team role feature for the PEM agents.                                                                                                                                                                                            |
```
Comment (sonotley) [inline product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx:18]: This is very vague. It's PEM-5109. Can we get a better release note from the eng team?

File: product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Postgres Enterprise Manager 9.6.0 release notes"
+navTitle: Version 9.6.0
+---
+
+Released: 15 May 2024
+
+New features, enhancements, bug fixes, and other changes in PEM 9.6.0 include:
+
+| Type         | Description                                                                                                                                                                                                                                         |
```
[the remark this answers is not in the document]
Comment (sonotley) [reply product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx:10]: In fact we seem to be missing a lot of release notes for big fixes - there are 61 Jiras in this release! But that's not a blocker for these docs we can review for next time

**Review by sonotley — CHANGES_REQUESTED** [review]

File: product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+| Bug&nbsp;fix | Fixed an issue with error `NoneType` has no `len()` when alerts were listed on the alert dashboard.                                                                                                                                                 |
+| Bug&nbsp;fix | Fixed an issue for team role feature for the PEM agents.                                                                                                                                                                                            |
+| Bug&nbsp;fix | Fixed an issue with SQL query for the PEM agent registration. Added the public schema name in the table prefix to execute the SQL query successfully even if schema name is not in the search path.                                                 |
+| Big&nbsp;fix | Updated the alert template `Table size in server/database/schema` with more detailed SQL query.                                                                                                                                                     |
```
[the remark this answers is not in the document]
Comment (nidhibhammar) [reply product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx:20]: I think we can write it as "Enhancement" if it is not a bugfix

File: product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (nidhibhammar) [reply product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx:10]: Ok, I will take care of it in separate PR

**Review by sonotley — APPROVED** [review]

**Pull request merged**